Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -114,9 +121,16 @@
                 </PaginationItem>
             }
             <PaginationItem IsDisabled="@(CurrentPage >= LastPage)">
-                <PaginationLink Page="next" Clicked="@OnPaginationItemClick">
-                    Next
-                </PaginationLink>
+              <PaginationLink Page="next" Clicked="@OnPaginationItemClick">
+                @if (NextButtonTemplate == null)
+                {
+                  <Label>Prev</Label>
```

Comment:
Should this be "Next"?

Reply:
You're right. It was a copy past misstake. I fixed it now.

Reply:
Hey, i could not start demo project. I get "RunCommand property not set" Message, when i want to start it with ISSExpess. Can you help me?

Reply:
Sorry, I dont know how to solve that :(

Reply:
Would you Test it for me? I tried it at my project, but i have problems with to many old references. I saw, that it works, but because of java script execption, it does not show my data.

Reply:
Ahhh, i'm idiot. xD I have not started server project. :)